### PR TITLE
Fix lists in v8.json

### PR DESCRIPTION
**Pull request by wipfli**
## Launch Checklist

 - [ ] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [ ] Briefly describe the changes in this PR.
 - [ ] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [ ] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

**Comment by HarelM** [comment]
Since we are talking about only 15 places, any chance you can manually check them?
I think `\n\n` is not needed between list items, but only before the first item on a list, so it might be that there are less places that actually need this fix.

File: src/reference/v8.json [hunk 1]
```diff
@@ -134,7 +134,7 @@
...
+      "doc": "A URL template for loading signed-distance-field glyph sets in PBF format. \n\nThe URL must include:\n\n - `{fontstack}` - When requesting glyphs, this token is replaced with a comma separated list of fonts from a font stack specified in the text-font property of a symbol layer. \n\n - `{range}` - When requesting glyphs, this token is replaced with a range of 256 Unicode code points. For example, to load glyphs for the Unicode Basic Latin and Basic Latin-1 Supplement blocks, the range would be 0-255. The actual ranges that are loaded are determined at runtime based on what text needs to be displayed.\n\nThis property is required if any layer uses the `text-field` layout property. The URL must be absolute, containing the [scheme, authority and path components](https://en.wikipedia.org/wiki/URL#Syntax).",
```
Comment (wipfli) [inline src/reference/v8.json:137]: Not sure Harel that we only need the double new lines for lists. For example this one here does not have any effect in the docs website if we only do one new line...

Comment (wipfli) [reply src/reference/v8.json:137]: image
https://maplibre.org/maplibre-style-spec/glyphs/

Comment (HarelM) [reply src/reference/v8.json:137]: Fair enough.

**Pull request merged**